ArDOS/ArdosRunner/DefaultRunner.cs

```diff
@@ -11,6 +11,8 @@ namespace ArDOS.Runner
 {
     public class DefaultRunner : IRunner
     {
+        public const string ARDOS_VERSION = "ArDOS_v1.0.0";
```

Comment:
Remove the `ArDOS_` part.

```suggestion
        public const string ARDOS_VERSION = "1.0.0";
```